tutorials/ten_minutes.ipynb

```diff
@@ -10,11 +10,11 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    sdf = spark.createDataFrame(input_df)
</pre><p>Kevin, can you explain the reason behind converting the pd.DataFrame to a sark dataframe before passing it to transform(). I thought transfrom() done this inherently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
It will but for a 10 minute tutorial, we want to also show you can do:

---
spark code
----
transform
----
spark code
-----


So it can play along with native Spark code. Using pandas throughout a 10 minute tutorial might imply the input needs to be pandas and we don't want that. Though I guess most people reading this won't be super Spark users. I don't know. I leave it up to your judgment.